code/modules/clothing/suits/wiz_robe.dm

```diff
@@ -84,11 +84,29 @@
 	flags_inv = HIDEHAIR|HIDEFACIALHAIR
 	dog_fashion = null
 
-/obj/item/clothing/head/wizard/hood
+/obj/item/clothing/head/hooded/wizard
```

Comment:
needs `clothing_flags = SNUG_FIT|CASTING_CLOTHES` or they won't work to cast spells with